commander_linux.yaml

```diff
@@ -37,4 +37,19 @@ tests:
     stdout:
       contains:
         - Use --dir to test directories with multiple test files
-    exit-code: 1
\ No newline at end of file
+    exit-code: 1
+
+  test nodes with stdin:
```

Comment:
I would prefer a extra test suite for this case which is simpler, i.e. with just an `echo hello` test. 
Some disadvantages about this approach:
 - Node implementation breaks but test for `stdin` are breaking too, this would be misleading
 - If the implementation of nodes changes this test needs adjustments too
 - If the original nodes test is edited this needs adjustments too
 - Testing two things in one test is considered bad practice (in most cases) and should be avoided